modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/CarouselView.tsx

```diff
@@ -3,14 +3,99 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import React from 'react';
+import React, {useState} from 'react';
+import {Card} from '../../../../../../../frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards';
+import './carousel.scss';
 
 const CarouselView = (props: any) => {
-
-	// eslint-disable-next-line no-console
-	console.log('props = ', props);
-
-	return <>Add real implementation here.</>;
+	const initialItems = Array.from({ length: 16 }, (_, index) => ({
+		url: `https://via.placeholder.com/800x500?text=Imagen+${index + 1}`,
```

Comment:
Even in samples, we should not link to real external sites. If you need images, some ship with DXP, as samples in document library. See how we use them in [ckeditor sample module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/partials/balloon.jsp#L23).